lambdas/db-migration/src/index.ts

```diff
@@ -28,7 +28,7 @@ const getSecretConnectionConfig = async (SecretId: string): Promise<Knex.PgConne
   }
   const dbAccessMeta = JSON.parse(response.SecretString);
 
-  ['host', 'user', 'password', 'database'].forEach((key) => {
+  ['host', 'username', 'password', 'database'].forEach((key) => {
```

Comment:
PR update broke this convention, updating to facilitate work on blocked tickets.  This change shouldn't impact existing functionality.